gen3-client/logs/scoreboard.go

```diff
@@ -22,8 +22,10 @@ func PrintScoreBoard() {
 		fmt.Println("\n\nSubmission Results")
 		w := tabwriter.NewWriter(os.Stdout, 0, 0, 0, ' ', tabwriter.Debug)
 		for i, score := range scoreBoard {
-			if i < len(scoreBoard)-1 {
-				fmt.Fprintf(w, "Finished with %d retry(ies) \t %d\n", i, score)
+			if i < 2 {
+				fmt.Fprintf(w, "Finished with %d retry \t %d\n", i, score)
+			} else if i < len(scoreBoard)-1 {
+				fmt.Fprintf(w, "Finished with %d retries \t %d\n", i, score)
```

Comment:
should it be like this? so that i==0 is included in the plural form :P
```
if i == 1 {
     fmt.Fprintf(w, "Finished with %d retry \t %d\n", i, score)
} else if i < len(scoreBoard)-1 { 
    fmt.Fprintf(w, "Finished with %d retries \t %d\n", i, score)
} else ...
```

Reply:
😮 didn't know should use plural with 0. Thanks! fixed in  https://github.com/uc-cdis/cdis-data-client/pull/50/commits/8ac6d83336ed26cf176af9563747602b20092a92